packages/angular/src/schematics/storybook-migrate-defaults-5-to-6/migrate-default-5-to-6.ts

```diff
@@ -0,0 +1,19 @@
+import { chain, externalSchematic, Rule } from '@angular-devkit/schematics';
+import { StorybookMigrateDefault5to6Schema } from './schema';
+import { wrapAngularDevkitSchematic } from '@nrwl/devkit/ngcli-adapter';
+
+export default function (schema: StorybookMigrateDefault5to6Schema): Rule {
```

Comment:
I noticed that for a few of these schematics, the folder is plural: 
        `migrate-defaultS-5-to-6` 
while the file names are not:
         `migrate-default-5-to-6`

If it won't affect the usage/confuse the users then it's probably not that important, and feel free to ignore 😄